examples/slack-starter/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "matrix-bridge-slack-starter",
+  "private": "true",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "start": "node ./lib/index.js",
+    "build": "tsc",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Matrix.org",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@types/node": "^14",
+    "@types/request": "^2.48.5",
+    "matrix-appservice-bridge": "file:../.."
```

Comment:
Not sure this will work for github starters, we might need to reference the actual package name or is GitHub that clever?

Reply:
Oh nvm, I thought you were creating a github starter project here.

Reply:
It's meant to be run in that folder. But it could become a starter, if people copy it and reference the NPM release.

Reply:
Tricky, because I can easily see people copying the project to a new directory and it falling apart. However, I'm slightly more concerned with having to update the version and keep it in line with the release, so let's use this for now.